File: Python/2153.py
import sys, math

f = lambda : sys.stdin.readline().rstrip()

dic = dict()

value = 1;
s = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"

for i in range(len(s)):
    dic[s[i]] = value;
    value += 1

s = f()
result = 0
for i in s:
    if i in dic:
        result += dic[i]

if result <= 2:
    print("It is a prime word.")
else:
    tf = True
    for i in range(2, int(math.sqrt(result)+1)):
        if result%i == 0:
            tf = False; break
    print("It is a prime word." if tf else "It is not a prime word.")

File: Java/_2153.java
package Algorithm.BaekJoon;

import java.io.*;
import java.util.*;

public class _2153 {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        Map<Character, Integer> map = new HashMap<>();

        int value = 1;
        for(char i = 'a'; i <= 'z'; i++){
            map.put(i, value++);
        }

        for(char i = 'A'; i <= 'Z'; i++){
            map.put(i, value++);
        }

        String str = br.readLine();

        int result = 0;
        for(char c : str.toCharArray()){

            if(map.containsKey(c)){

                result += map.get(c);
            }
        }

        if(result <= 2){

            System.out.println("It is a prime word.");
        } else {

            boolean tf = true;
            double sqrtNum = Math.sqrt(result);
            for(int i = 2; i <= sqrtNum; i++){

                if(result%i == 0){

                    tf = false;
                    break;
                }
            }

            System.out.println(tf ? "It is a prime word." : "It is not a prime word.");
        }
    }
}
